app/src/main/res/values/strings.xml

```diff
@@ -1,4 +1,5 @@
 <resources>
     <string name="app_name">IBDB</string>
     <string name="title_activity_greeting">GreetingActivity</string>
+    <string name="add_new_book">Add a new book</string>
```

Comment:
Nitpick: As Cirrus indicates, this resource is unused. Might be interesting to just remove it until it is needed, it doesn't look like it would take too much time to put it again when needed.

Reply:
I figured we might still use it in the future but we can remove it for now, yes.